<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Random</class>
 <widget class="QDialog" name="Random">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="main_window_gridLayout" rowstretch="1,0,0,0,0" columnstretch="1,1">
     <item row="2" column="0" colspan="2">
      <widget class="QTableWidget" name="trials_tableWidget">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;</string>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>75</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>75</number>
       </attribute>
       <column>
        <property name="text">
         <string>Trial name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Parameters</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Precent in session</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="total_num_of_trials_spinBox"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="explenation_label">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Configurations</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="total_num_of_trials_label">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>Total number of trials:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Random</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Random</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
